doc/system_specific_setup.rst

```diff
@@ -50,6 +56,41 @@ pyparsing is required by PSyclone
 
    > sudo apt-get install python-pyparsing
 
+.. _install_pip:
+
+Install pip
+^^^^^^^^^^^
+
+The pip tool enables Python packages to be installed from the Python
+Package Index (https://packaging.python.org/installing/). Install it like so:
+::
+     > sudo apt-get install python-pip
+
+.. _install_fparser:
+
+Install fparser
+^^^^^^^^^^^^^^^
+
+fparser is also required by PSyclone but is not available from the
+Ubuntu software centre. It can instead be installed from the
+Python Package Index using pip:
+::
+
+    > pip install --user fparser
```

Comment:
Show how to install system wide first saying if you have appropriate permissions, then user then how to uninstall.